### ISSUE TITLE
Just installed Buoy now . Need further help.

### ISSUE THREAD

**Issue by ajaypandya1** [betterangels/buoy#135]
I just now installed the Buoy .
1) How will my future and present members access the Buoy on my website. ?
2) How will I create a Menu  tab on my website for members to access .
3). Do I need to create a page . If yes where will I get the short code ? 
As of these few first concerns .
A.J.
- Platform:  / Windows / 
- Browser:  / Mozilla Firefox / 
- URL: http://www.alertcitizenstoday.com

**Comment by fabacab** [comment]
Consider reading the [Buoy Walkthrough](https://github.com/betterangels/buoy/wiki/Buoy-Walkthrough) to learn how to get started.

Also, questions about Buoy's end-user use should be asked at the [WordPress Plugin Users' Support Forum](https://wordpress.org/support/plugin/buoy), not really here. This issue tracker is for work on the plugin itself.

That said:

> I just now installed the Buoy .
> 1) How will my future and present members access the Buoy on my website. ?

Registered users (users with a local WordPress user account) will see a "My Teams" menu item on the Dashboard Menu. There will also be "Crisis Response Team" and "Alert" items under the New Content menu.

> 2) How will I create a Menu tab on my website for members to access .

You don't need to create a menu, though of course you can. Users can access the Buoy teams, settings, and alerts options when they are logged in to the site much as they would for any other content type.

> 3). Do I need to create a page . If yes where will I get the short code ?
> As of these few first concerns .
> A.J.

You can, but you don't have to. Buoy does not add pages nor other items to the front-end of your public website at all, because it's not needed. If you want to advertise that your site is using Buoy and offers crisis response tools to its users, you are welcome to do this however you wish, and you would do so just as you might write any other content on the website.

**Comment by ajaypandya1** [comment]
Hello ,          
Hello ,
 I as an admin can see Bouy in my admin Dashboard , But my users do not see the Bouy Menus like Add my Teams , My Teams etc. 

In short how will my users access Bouy . I can excees in my admin dashboard
What should I do           AJAY.

```
On Thursday, June 9, 2016 2:33 PM, Meitar M. <notifications@github.com> wrote:
```

 Closed #135.—

**Comment by fabacab** [comment]
> my users do not see the Bouy Menus like Add my Teams , My Teams etc.

Where are they looking?

Users need to log in to the website with their own account first, and then the My Teams, etc. menu will be shown to them in the same place as it is shown to you.
